README.md

```diff
@@ -9,14 +9,53 @@ which incidents. Its deployment happens once a day via
 
 ## Getting Started
 
-  To get started all you need is an empty PostgreSQL database and the following dependencies:
+### Install Dependencies
 
-    $ sudo zypper in -C postgresql-server postgresql-contrib
-    $ sudo zypper in -C perl-Mojolicious perl-Mojolicious-Plugin-Webpack \
-      perl-Mojo-Pg perl-Cpanel-JSON-XS perl-JSON-Validator perl-IO-Socket-SSL nodejs16
-    $ npm install
+To install all required dependencies, run
 
-Update the config file `dashboard.yml` to point to your PostgreSQL database (and other services where appropriate):
+    sudo zypper in -C postgresql-server postgresql-contrib
+    sudo zypper in -C perl-Mojolicious perl-Mojolicious-Plugin-Webpack \
+      perl-Mojo-Pg perl-Cpanel-JSON-XS perl-JSON-Validator perl-IO-Socket-SSL nodejs18
+    npm install
+
+if you are on an apt-based system, run 
+
+    npx playwright install --with-deps
+
+if you are on a zypper-based system, omit the `--with-deps` parameter
+
+    npx playwright install
+
+and ignore the missing dependencies
+
+#### nodejs18
+
+In case `nodejs18` is not available (e.g. Tumbleweed), you can alternatively get it via the [Node Version Manager](https://github.com/nvm-sh/nvm). Simply [install nvm](https://github.com/nvm-sh/nvm?tab=readme-ov-file#install--update-script) then run `nvm install 18` and `nvm use 18`.
+
+### Postgres Database
+
+The postgresql-server package installation in the previous step, created a database called `postgres` by the user `postgres`.
+
+Allow this user to connect to the local postgres server by modifying `/var/lib/psgql/data/pg_hba.conf`
+
+```
+# TYPE  DATABASE        USER            ADDRESS                 METHOD
+# "local" is for Unix domain socket connections only
+local   all             all                                     reject
+# IPv4 local connections:
+host    postgres        postgres        127.0.0.1/32            trust
+host    all             all             127.0.0.1/32            reject
+# IPv6 local connections:
+host    postgres        postgres        ::1/128                 trust
+host    all             all             ::1/128                 reject
+# Allow replication connections from localhost, by a user with the
+# replication privilege.
+local   replication     all                                     reject
+host    replication     all             127.0.0.1/32            reject
+host    replication     all             ::1/128                 reject
```

Comment:
Do all of those lines need changes or can you focus on just the parts that needed changes?

Reply:
Yes and no, i googled and it seems that the default `METHOD` values are not consistent, which is why i just set all defaults to `reject` and added only the specific permissions needed for running the qem-dashboard (line 46 and 49). Its also important that the settings are in the correct order. So in theory adding these two lines would be enough, though less secure.